ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v2/aemform/clientlibs/editorhook/js/EditListeners.js

```diff
@@ -0,0 +1,61 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+(function (Granite) {
```

Comment:
Can you change this as per this, https://github.com/adobe/aem-core-wcm-components/tree/a7e377c6810dd2125a902d3ef3df5c92ec2a1719/content/src/content/jcr_root/apps/core/wcm/components/experiencefragment/v2/experiencefragment/clientlibs/editorhook/js. Please add readable namespace, let's not use window directly

Reply:
Let's add all of this in namespace `window.CQ.FormsCoreComponents`. Your code should be part of `window.CQ.FormsCoreComponents.aemform.v2.actions`

Reply:
Done